flink-python/pyflink/table/tests/test_row_based_operation.py

```diff
@@ -189,6 +191,7 @@ def test_aggregate_with_pandas_udaf_without_keys(self):
         actual = source_sink_utils.results()
         self.assert_equals(actual, ["+I[3.8, 8]"])
 
+    @unittest.skip("Not supported yet")
```

Comment:
This is a bug in the batch time window of the table api. In the new type system in this case, the type of the window reference will be `localDateTime`